packages/react/tooltip/src/tooltip.tsx

```diff
@@ -303,6 +303,7 @@ const TooltipTrigger = React.forwardRef<TooltipTriggerElement, TooltipTriggerPro
             hasPointerMoveOpenedRef.current = false;
           })}
           onPointerDown={composeEventHandlers(props.onPointerDown, () => {
+            context.onClose();
```

Comment:
I think we should call this conditionally in the event that the user's handler triggers some sort of side-effects or unrelated state changes.
```suggestion
            if (context.open) {
              context.onClose();
            }
```